Fix heading styles in Glossary Definition and Tooltip

pettinarip opened the pull request:
In recent changes, we have changed a bit how we display the heading on the Glossary Definition component, and this affects the Glossary Tooltip styles.

## Description

This PR fixes the style issues by:
- using the new [Heading](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Heading/index.tsx) component with DS styles
- removing the GlossaryDefinition dependency on GlossaryTooltip since they look quite different now (no more reusability here)

Bonus: added a new story for the open state of GlossaryTooltip to avoid getting these regressions again.

TylerAPfledderer commented on src/components/Glossary/GlossaryTooltip/GlossaryTooltip.stories.tsx:
```diff
@@ -5,13 +5,21 @@ import GlossaryTooltipComponent from "."
+export const Basic: Story = {}
+
+// for chromatic story snapshot showing the rendered popover
+export const OnOpen: Story = {
```
Need to add a decorator here, where it provides enough white-space to cover the visible tooltip. Otherwise the tooltip will be cropped out in the chromatic snapshot.

The decorator can be supplied at the individual story level. (There is currently a decorator at the component level for the [Tooltip](https://github.com/ethereum/ethereum-org-website/blob/dev/src%2Fcomponents%2FTooltip%2FTooltip.stories.tsx#L46-L52) for this very reason)

TylerAPfledderer reviewed (changes requested):
@pettinarip quick thought from me 😁

corwintines commented:
@pettinarip @TylerAPfledderer I think this will be ready now

I added the `glossary-tooltip` namespace into `.storybook/i18next.ts`, and changed the term from `big-endian` to `bridge` since `big-endian` isn't in glossary-tooltip.json.

wackerow commented:
@nloureiro Mind approving these changes in Chromatic if they look okay to you?

https://www.chromatic.com/review?appId=63b7ea99632763723c7f4d6b&number=13205&type=linked&view=changes

pettinarip commented:
> @pettinarip @TylerAPfledderer I think this will be ready now
> 
> I added the `glossary-tooltip` namespace into `.storybook/i18next.ts`, and changed the term from `big-endian` to `bridge` since `big-endian` isn't in glossary-tooltip.json.

Nice! thanks!

nloureiro commented:
> @nloureiro Mind approving these changes in Chromatic if they look okay to you?
> 
> https://www.chromatic.com/review?appId=63b7ea99632763723c7f4d6b&number=13205&type=linked&view=changes

looks good to me. approved on Chromatic :)
